pom.xml

```diff
@@ -64,13 +64,13 @@
     <trimStackTrace>false</trimStackTrace> <!-- SUREFIRE-1226 workaround -->
     <!-- Whether the build should fail if findbugs finds any error. -->
     <!-- It is strongly encouraged to leave it as true. Use false with care only in transient situations. -->
-    <findbugs.failOnError>true</findbugs.failOnError>
+    <spotbugs.failOnError>true</spotbugs.failOnError>
```

Comment:
Could it default to the original constant name so that it is compatible?

Reply:
I'll keep the naming did not think of the POMs that might be using the variables.